build/CoreFoundation/dll/CoreFoundation.def

```diff
@@ -29,6 +29,8 @@ LIBRARY CoreFoundation
         _CFAppendPathExtension2
         _CFApplicationPreferencesCopyRepresentation
         _CFRuntimeBridgeTypeToClass
+		__CFISAForTypeID
```

Comment:
oops, saw this earlier, thought I had fixed it

---
In reply to: [78851772](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r78851772) [](ancestors = 78851772)